daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/TypeCalculator.scala

```diff
@@ -153,7 +153,7 @@ class KeysetValueTypeCalculatorOrdered(valueMap: HashMap[DataValuePrimitive, Dat
       })
       ans1 match {
         case None => {
-          (DataValue.NoValue, One(s"Key ${x} not found in keyset-value mapping"))
+          (DataValue.NoValue, One(s"Key ${x} not found in enumeration key-value mapping"))
```

Comment:
I guess we were missing this coverage in the past, but it might be nice to add additional tests to improve coverage of all these different error conditions.